src/runtime_handlers.cpp

```diff
@@ -44,6 +46,11 @@ namespace hpx { namespace detail {
         std::abort();
     }
 
+    bool enable_parent_task_handler()
+    {
+        return (hpx::get_initial_num_localities() == 1);
+    }
+
```

Comment:
Should this be `ifdef`'ed for `HPX_HAVE_APEX`?

Reply:
Yep, good catch.